android/src/main/java/com/mattermost/networkclient/APIClient.kt

```diff
@@ -26,7 +26,7 @@ class APIClientModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
             sessionsClient[baseUrl]!!.parseOptions(options);
 
             // Return stringified client for success
-            promise.resolve(Unit)
+            promise.resolve( null)
```

Comment:
I thought `Unit` = void? what happened here?

Reply:
I kept having some issues with the Java `Void` class getting initialised here (and attempts to send this class over the bridge in the `promise.resolve`), when I really just wanted the `void` type - but weirdly they both ended up coming across as `null` over the bridge so figured `null` could just do rather than digging too deep into the Kotlin `Unit` >> Java `Void` interop